pymine_net/net/asyncio/client.py

```diff
@@ -0,0 +1,34 @@
+import asyncio
+from typing import Union
+
+from pymine_net.net.asyncio.stream import AsyncTCPStream
+from pymine_net.net.client import AbstractTCPClient
+from pymine_net.types.packet import ClientBoundPacket, ServerBoundPacket
+from pymine_net.types.packet_map import PacketMap
+
+__all__ = ("AsyncTCPClient",)
+
+
+class AsyncTCPClient(AbstractTCPClient):
+    """An async connection over a TCP socket for reading + writing Minecraft packets."""
+
+    def __init__(self, host: str, port: int, protocol: Union[int, str], packet_map: PacketMap):
+        super().__init__(host, port, protocol, packet_map)
+
+        self.stream: AsyncTCPStream = None
+
+    async def connect(self) -> None:
+        _, writer = await asyncio.open_connection(self.host, self.port)
```

Comment:
What is the purpose of the underscore here? I don't see it referenced again.

Reply:
Using a single underscore is convention for when you need to assign something but not use it. In this case `asyncio.open_connection(...)` returns a tuple and I only need the second value from the tuple. Other common uses are in for loops like `for _ in range(...): ...`